### ISSUE TITLE
ffmpeg error, code 127

### ISSUE THREAD

**Issue by Rubinat** [nypublicradio/audiogram#31]
Hello,

My apologies if I am asking something stupid. I am not that familiar with programming in general; The same going for npm, brew, ffmpeg, nodejs etc.
Also please forgive me for any bad English as it is not my native language. :)

At the moment I am working on an iMac running El Capitan and am trying to get the audiogram server and client to work.
So far I have managed to get the terminal to go into listening mode, and then I was able to reach the client through my browser. It seemed to work properly up until I pressed the Generate button.
At that point the terminal spit out an error and the browser showed a continuous loading screen.

I have been trying to find a solution to get the error to disappear and the app to start working, but I am afraid that I can't find the information I am looking for.
Maybe because I am making a beginners mistake, or maybe I haven't got enough knowledge to proceed with the other tips I have found online.

Therefor I am hoping that someone here would be able and willing to help me out.
I tried to follow the step as properly as I could.
The error I got when pressing the Generate button in the browser was:

`{"status":"error","error":{"killed":false,"code":127,"signal":null,"cmd":"ffmpeg -i /PATH/TO/DIRECTORY/audiogram/tmp/7551fab0-5988-11e6-9be1-5b8c780d4c8d/audio -ss 0 -t 29.585714328125 -acodec libmp3lame -b:a 128k //PATH/TO/DIRECTORY/audiogram/tmp/7551fab0-5988-11e6-9be1-5b8c780d4c8d/audio-trimmed.mp3"}}`

So far I have tried updating all the dependencies/libs, also uninstalling and re-installing them, uninstalling ffmpeg only (because it might work with libgroove only), and tried running it and running it as root.

If anyone could help me out that would be greatly appreciated!
Greetings,
Ruben

**Comment by veltman** [comment]
You've left the string `/PATH/TO/DIRECTORY/` in your workingDirectory in `settings/index.js`.  You should replace that with the path to the actual Audiogram directory.

**Comment by Rubinat** [comment]
Hello,

Thank you for the fast reply! :)
The `PATH/TO/DIRECTORY` is something I filled in myself when I posted the question (to keep a little privacy).
The place where is now `PATH/TO/DIRECTORY` actually reads the correct path. At least I think I am fairly certain it does.

Do I still need to fill in an absolute path into the `settings/index.js`?

Thanks again for helping out so quickly!
Greetings,
Ruben

**Comment by Rubinat** [comment]
Hello,

Thank you for the fast reply! :)
The `PATH/TO/DIRECTORY` is something I filled in myself when I posted the question (to keep a little privacy).
The place where is now `PATH/TO/DIRECTORY` actually reads the correct path. At least I think I am fairly certain it does.

Do I still need to fill in an absolute path into the `settings/index.js`?

Thanks again for helping out so quickly!
Greetings,
Ruben

**Comment by Rubinat** [comment]
Hello again,

I am not sure if you are still getting these messages, but I wanted to let you know that I managed to find a solution!
In the actual path there was a directory with multiple spaces and an `&`. Then I changed the directory name to one word and after that the app started working.

So I haven't checked it, but I assume the error was created by the spaces, the `&` or the combination of both.

Anyway it seems to be working fine now and I am fiddling around with the themes.

Thanks for your help!
Greetings,
Ruben
